src/pages/_app.js

```diff
@@ -58,38 +57,21 @@ function App({ Component, pageProps, renderer = fallbackRenderer }) {
       </Head>
       <Sprite />
 
-      <PlausibleProvider
-        domain='stormbound-kitty.com'
-        enabled={process.env.NEXT_PUBLIC_VERCEL_ENV !== 'preview'}
```

Comment:
Vercel Analytics knows which environment the site runs on and can segment the tracking per environment in the UI.